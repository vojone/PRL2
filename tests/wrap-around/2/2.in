1111000000000000000000000000000000000000
1101000000000000000000000000000000000000
1001000000000000000000000000000000000000
1111000000000000000000000000000000000001
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0001000000000000000000000000000000000000
