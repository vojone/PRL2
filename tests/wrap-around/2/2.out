0001100000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
1000000000000000000000000000000000000001
1000000000000000000000000000000000000001
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0000000000000000000000000000000000000000
0001100000000000000000000000000000000000
